core/server/worker/src/main/java/alluxio/worker/grpc/GrpcExecutors.java

```diff
@@ -48,6 +48,11 @@
           THREAD_STOP_MS, TimeUnit.MILLISECONDS, new SynchronousQueue<>(),
```

Comment:
If it is changed to `new LinkedBlockingQueue<>()`,  the mDataReaderExecutor#submit will not throw a `RejectedExecutionException` exception .

Reply:
I see, thanks for clarifying